data/sandramsc.json

```
@@ -4,22 +4,26 @@
   "tags": [
     "React.js",
     "Python",
-    "Typescript",
-    "WordPress",
     "Open Source",
     "Artificial Intelligence (AI)",
+    "WordPress",
     "Web3"
   ],
   "socials": [
     {
       "icon": "FaLinkedin",
-      "color": "#0000FF",
+      "color": "#000000",
       "url": "https://linkedin.com/in/sandraashipala"
     },
     {
       "icon": "FaGithub",
```

[Proofreading] See [my feedback on this line](https://github.com/EddieHubCommunity/LinkFree/pull/7706/files#r1235947915). 
```suggestion
      "icon": "FaGithub"
```

---

@CBID2 please see comment which applies too: https://github.com/EddieHubCommunity/LinkFree/pull/7706/files#r1236225739

I will mark as resolved.